widget/widgetAssets/html/timSchedulePlaybook.html

```diff
@@ -0,0 +1,127 @@
+<!-- Copyright start
+  MIT License
+  Copyright (c) 2024 Fortinet Inc
+  Copyright end -->
+<form data-ng-submit="save()" class="noMargin" name="scheduleForm" data-ng-class="{'state-wait': processing }"
+  novalidate>
+  <fieldset data-ng-disabled="!allowEdit">
+    <div class="row">
+      <div data-ng-if="!params.form.schedule.kwargs.exit_if_running"
+        class="col-md-12 font-size-14 margin-bottom-md padding-0">Do you want to schedule the ingestion?<span
+          data-ng-show="field.required && form.config.ignoreRequired" class="text-danger">*</span></div>
+      <div data-ng-if="!params.form.schedule.kwargs.exit_if_running" class="cs-select width-150 float-left">
+        <select class="form-control width-150" data-ng-model="params.form.scheduled">
+          <option value="N">No</option>
+          <option value="Y">Yes</option>
+        </select>
+        <span class="fa fa-sort-desc"></span>
+      </div>
+      <div class="col-md-12 padding-left-0 margin-top-md">
+        <h6 class="control-label display-block margin-0 margin-bottom-md">Configure Schedule Settings</h6>
```

Comment:
all these hard coding can be derived from consts

Reply:
will this change also help in supporting i18n?

Reply:
File Removed